<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuxgateOpenOrdersPanel</class>
 <widget class="QFrame" name="LuxgateOpenOrdersPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTotalTytle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TOTAL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBaseTotalCurrency">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBaseTotal">
       <property name="text">
        <string>34.5566</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelQuoteTotalCurrency">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelQuoteTotal">
       <property name="text">
        <string>134</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableViewOpenOrders"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAllCancel">
       <property name="text">
        <string>Cancel All</string>
       </property>
       <property name="icon">
        <iconset resource="../lux.qrc">
         <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../lux.qrc"/>
 </resources>
 <connections/>
</ui>
